packages/async-rewriter2/package.json

```diff
@@ -28,13 +28,10 @@
   },
   "dependencies": {
     "@babel/core": "^7.9.0",
-    "@babel/parser": "^7.9.4",
     "@babel/plugin-transform-destructuring": "^7.13.0",
     "@babel/plugin-transform-parameters": "^7.13.0",
     "@babel/plugin-transform-shorthand-properties": "^7.12.13",
-    "@babel/traverse": "^7.9.0",
     "@babel/types": "^7.9.0",
-    "@types/babel__core": "^7.1.6",
-    "@types/babel__traverse": "^7.0.9"
+    "@types/babel__core": "^7.1.6"
```

Comment:
I think you’d also want to run `npm bootstrap` and commit the changed package-lock.json files?